docs/rules/no-invalid-model-keys.md

```diff
@@ -0,0 +1,108 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-invalid-model-keys
+description: require valid keys in model prop
+since: v3.9.0
```

Comment:
`since` is automatically set at the time of release. Would you please remove this.

```suggestion
```